core/lib/types/src/commitment/mod.rs

```diff
@@ -401,11 +430,38 @@ impl L1BatchAuxiliaryOutput {
                     aux_commitments,
                     blob_linear_hashes,
                     blob_commitments,
+                    local_root,
+                    aggregated_root,
                 }
             }
         }
     }
 
+    pub fn get_local_root(&self) -> H256 {
```

Comment:
Bikeshedding: Although this naming is used in the codebase already, `get_` prefix [is considered](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter) redundant in Rust.